src/components/primitive/button/button.component.styles.ts

```diff
@@ -1,27 +1,58 @@
 import styled from 'styled-components';
-import { StyledButtonProps } from './button.component.props';
-import {
-  handleBorderColors,
-  handleActiveColors,
-  handleDefaultBackground,
-  handleDefaultButtonText,
-} from './button.component.utils';
+import { Box } from '../box/box.component';
+import { Text } from '../text/text.component';
 
-export const StyledButton = styled.button<StyledButtonProps>`
-  padding: 1rem 2rem;
-  border-radius: 4px;
+/**
+ * Styles for Primary Button.
+ */
+export const Primary = styled(Box)`
```

Comment:
- Naming is not clear. `Primary` name does not indicate that it's a button. You can name it PrimaryButton or so.
- Also primary seems to have a lot of styles. There must be a BaseButton with common styles which you can extend to create other buttons.